src/huggingface_hub/inference/_providers/__init__.py

```diff
@@ -111,6 +113,10 @@
         "conversational": NovitaConversationalTask(),
         "text-to-video": NovitaTextToVideoTask(),
     },
+    "nscale": {
+        "conversational": NscaleChatCompletion(),
```

Comment:
```suggestion
        "conversational": NscaleConversationalTask(),
```
(related to my comment in `src/huggingface_hub/inference/_providers/nscale.py`.